_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+---
+id: 825ule
+name: Non-visual location reference alternative
+rule_type: atomic
+
+description: |
+  This rule checks that when there is a references to a visual location of content, there are also non-visual indicators of the location.
+
```

Comment:
I'm a bit confused :-) 

The rule title is "non-visual **location** reference". *Location* is one type  of visual reference (as the G96 talks about *"shape, size, sound or locations"*). The rule seems to focused only on *location*, and that's fine. 

But the in the applicability, we talk about "one of the visual reference words". 

I think the rule focused to every visual references (see passed example 6 "The button is indicated by the word "round". But the word is also included in the text of the element." is about "shape" not location).

So rename the rule and rewrite the description?

Proposal

>  name: Non-visual reference alternative

> description: |
  This rule checks that when there is a visual reference of content, there are also non-visual indicators of the location.